main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"code-review-bot/internal"
+	"code-review-bot/internal/web"
+	"code-review-bot/internal/web/routes"
```

Comment:
Evaluate whether creating an HTTP client with `httpcache.NewMemoryCacheTransport()` is suitable for your use case. Memory caching may not be ideal for production environments if you expect high traffic, as it can lead to excess memory usage. Consider configuring it for disk caching or reviewing the caching strategy based on the application load and behavior.